.github/workflows/release.yml

```diff
@@ -305,6 +316,62 @@ jobs:
           tags: |
             public.ecr.aws/covalent/covalent:latest
             public.ecr.aws/covalent/covalent:${{ env.TAG }}
+
+      - name: Download Covalent AWS executor Dockerfile
+        id: get_aws_executor_dockerfile
+        run: |
+          url="https://raw.githubusercontent.com/AgnostiqHQ/covalent-aws-plugins/develop/Dockerfile?token=${{ secrets.COVALENT_OPS_BOT_TOKEN }}"
+          wget $url -O Dockerfile.awsexecutor
```

Comment:
This will fail the whole workflow if the Dockerfile doesn't exist. I think we need to check for `status_code` first, and then only get the file if `status_code -lt 400`